app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -54,6 +71,45 @@ def grant_admin_set_access(manage_grants)
           admin_set.edit_groups = manage_grants.select { |x| x[:agent_type] == 'group' }.map { |x| x[:agent_id] }
           admin_set.save!
         end
+
+        # Select appropriate radio button under Release "Varies" option based on saved permission_template
+        def select_release_varies_option(permission_template)
+          # Ignore 'no_delay' or 'fixed' values, as they are separate options
+          unless permission_template.release_period == Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_NO_DELAY ||
+                 permission_template.release_period == Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_FIXED
```

Comment:
I might like to see this and similar logic moved into the PT itself, so you can ask it questions like `permission_template.release_varies?` or `permission_template.release_before?`